app/forms/new_phone_form.rb

```diff
@@ -130,12 +130,9 @@ def validate_not_premium_rate
   end
 
   def validate_recaptcha_token
-    return if !validate_recaptcha_token? || recaptcha_validator.valid?(recaptcha_token)
-    errors.add(
-      :recaptcha_token,
-      I18n.t('errors.messages.invalid_recaptcha_token'),
-      type: :invalid_recaptcha_token,
-    )
+    return if !validate_recaptcha_token?
+    recaptcha_validator.submit(recaptcha_token)
+    errors.merge!(recaptcha_validator)
```

Comment:
does `errors.merge!` work with an entire validator? I'd expect more like
```suggestion
    recaptcha_validator.submit(recaptcha_token)
    errors.merge!(recaptcha_validator.errors)
```

Reply:
That's what I initially expected as well, but it error'd because `ActiveModel::Errors` doesn't have an `errors` method; it's expecting the model, not the errors.